lib/node_modules/@stdlib/math/base/special/csignum/benchmark/benchmark.js

```diff
@@ -90,7 +91,7 @@ bench( pkg+'::manual', function benchmark( b ) {
 	for ( i = 0; i < b.iterations; i++ ) {
 		re = ( randu()*1000.0 ) - 500.0;
 		im = ( randu()*1000.0 ) - 500.0;
-		az = cabs( re, im );
+		az = cabs( new Complex128( re, im ) );
```

Comment:
This is fine for now. We'll want to refactor once `csignum` is updated, as these benchmarks are now more likely to measure complex number creation, than the perf of the operation.